addons/sys_bf888s/RadioDialogClasses.hpp

```diff
@@ -0,0 +1,79 @@
+class Prc343_RscBackground
+{
+    type = CT_STATIC;
+    idc = -1;
+    style = ST_BACKGROUND + ST_KEEP_ASPECT_RATIO;
+    colorBackground[] = {0, 0, 0, .75};
+    colorText[] = {1, 1, 1, 1};
+    font = FontM;
+    sizeEx = 0.04;
+};
+
+// text
+class Prc343_RscText
+{
+    idc = -1;
+    type = CT_STATIC;
+    style = ST_LEFT + ST_KEEP_ASPECT_RATIO;
+    font = EtelkaMonospaceProBold;
```

Comment:
Quote.

Reply:
Not entirely sure what you mean by this one.